funding/manager.go

```diff
@@ -3459,6 +3670,17 @@ func (f *Manager) handleFundingLocked(peer lnpeer.Peer,
 		return
 	}
 
+	// If this is a taproot channel, then we can generate the set of nonces
+	// the remote party needs to send the next remote commitment here.
+	var firstVerNonce *musig2.Nonces
+	if channel.ChanType.IsTaproot() {
+		firstVerNonce, err = genFirstStateMusigNonce(channel)
```

Comment:
nit: guess we could use `channel.GenMusigNonces` here since the `lnChan` has already been populated, future refactoring stuff I guess.

Reply:
At this point, we don't have the lnwallet channel yet.